vnext/ReactWindowsCore/Modules/AppThemeModule.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "AppThemeModule.h"
+
+namespace react { namespace windows {
+
+//
+// AppTheme
+//
+
+AppTheme::AppTheme() = default;
+AppTheme::~AppTheme() = default;
+
+// TODO: real implementation
+const std::string AppTheme::getCurrentTheme()
+{
+  return "light";
+}
+
+//
+// AppThemeModule
+//
+
+/* static */ const std::string AppThemeModule::name = "RTCAppTheme";
+
+AppThemeModule::AppThemeModule(std::shared_ptr<AppTheme>&& appTheme)
```

Comment:
Not sure you need the whole AppThemeModule + AppThemeModuleUwp - its fine but we had that in appstate and some other modules to support win32 plugging in a different implementation when they wanted the same module, not sure they'll need this one.

Reply:
Discussed offline. Will leave it as is for now. We did do it this way thinking win32 might want to use it, but it seems they have their own theming solution.